pegasus/sites.v3/hourofcode.com/styles/060-hoc-specific.css

```diff
@@ -216,3 +216,11 @@ iframe {
     margin-left: 1em;
   }
 }
+.hidden-label {
+  position:absolute;
```

Comment:
NIT: May be preferred to use `position: fixed` rather than absolute. It's unlikely, but possible that someone may try to solve the same issue in the future for another element on the page with `left:10000px` which could cause this to show up again.